pkg/gui/presentation/branches.go

```diff
@@ -36,7 +37,9 @@ func getBranchDisplayStrings(b *models.Branch, fullDescription bool, diffed bool
 	}
 	coloredName := nameTextStyle.Sprint(displayName)
 	if b.IsTrackingRemote() {
-		coloredName = fmt.Sprintf("%s %s", coloredName, ColoredBranchStatus(b))
+		if !b.Gone || goneBranchText != "" {
```

Comment:
are these checks necessary?

Reply:
Yeah, this is because I don't want a whitespace after the branch name if the text is `""`, visually this makes no difference and since the text is not configurable anymore, it makes sense to remove the checks.